src/box/index_def.c

```diff
@@ -89,6 +90,7 @@ const struct opt_def index_opts_reg[] = {
 	OPT_DEF("func", OPT_UINT32, struct index_opts, func_id),
 	OPT_DEF_LEGACY("sql"),
 	OPT_DEF_CUSTOM("hint", index_opts_parse_hint),
+	OPT_DEF("fast_offset", OPT_BOOL, struct index_opts, fast_offset),
```

Comment:
If we decide to keep the flag, let's enable it by default, as it was suggested in the design document.

Reply:
It's been decided to only keep one version of the tree (the one with cardinality info enabled).